interfaces/repo.go

```diff
@@ -655,6 +655,32 @@ func (r *Repository) RemoveSnap(snapName string) error {
 	return nil
 }
 
+// AutoConnectBlacklist returns plugs that should not be auto-connected.
+//
+// Plug is blacklisted if it has no connections despite using an auto-connected
+// interface. That implies it was manually disconnected after installation.
```

Comment:
s/after installation//

We'll likely support blacklisting _during_ installation at some point.